.travis.yml

```diff
@@ -1,27 +1,31 @@
-sudo: required
+dist: trusty
+sudo: false
+
 language: scala
 scala:
 - 2.11.7
-services:
-  - docker
-before_install:
-- docker pull java:8u92-jdk-alpine
-- wget https://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/0.13.11/sbt-launch.jar
-- echo 'java -Xms512M -Xmx1536M -Xss1M -XX:+CMSClassUnloadingEnabled -XX:MaxPermSize=256M -jar sbt-launch.jar "$@"' > sbt.sh
+jdk:
+- oraclejdk8
+
+cache:
+  # md5deep
+  # - https://github.com/travis-ci/travis-ci/issues/3122
+  # - https://github.com/travis-ci/travis-ci/issues/3039
+  branch: md5deep
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot/
+before_cache:
+  # Tricks to avoid unnecessary cache updates
+  - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
+  - find $HOME/.sbt -name "*.lock" -delete
```

Comment:
It's still required to improve the stability of the build - especially if a build fails.